### PR TITLE
펫 추가 등록 기능 구현

### PR DESCRIPTION
**Pull request by naekang**
## Description

> 펫 추가 등록 기능 구현

## Progress

- [x] 펫 추가 기능
- [x] 테스트코드 작성

### CODE REVIEW

**Comment by naekang** [comment]
|File|Coverage [82.01%]|:green_apple:|
|:-|:-:|:-:|
|[PetErrorCode.java](https://github.com/Gaethering/Gaethering-Server/blob/40c630b812201f95981db79b7230d89bb8b76c7d/src%2Fmain%2Fjava%2Fcom%2Fgaethering%2Fgaetheringserver%2Fpet%2Fexception%2Ferrorcode%2FPetErrorCode.java)|96.3%|:green_apple:|
|[PetServiceImpl.java](https://github.com/Gaethering/Gaethering-Server/blob/40c630b812201f95981db79b7230d89bb8b76c7d/src%2Fmain%2Fjava%2Fcom%2Fgaethering%2Fgaetheringserver%2Fpet%2Fservice%2FPetServiceImpl.java)|78.57%|:x:|
|[PetController.java](https://github.com/Gaethering/Gaethering-Server/blob/40c630b812201f95981db79b7230d89bb8b76c7d/src%2Fmain%2Fjava%2Fcom%2Fgaethering%2Fgaetheringserver%2Fpet%2Fcontroller%2FPetController.java)|67.5%|:x:|

|Total Project Coverage|48.81%|:x:|
|:-|:-:|:-:|

**Review by bokyung95 — COMMENTED** [review]
빠르게 잘 작성해주셨네요! 수고하셨습니다👍

**Comment by naekang** [comment]
|File|Coverage [82.01%]|:green_apple:|
|:-|:-:|:-:|
|[PetErrorCode.java](https://github.com/Gaethering/Gaethering-Server/blob/249c6c96eb36759a8c96bfa6f2ab4d594ffa2dc5/src%2Fmain%2Fjava%2Fcom%2Fgaethering%2Fgaetheringserver%2Fpet%2Fexception%2Ferrorcode%2FPetErrorCode.java)|96.3%|:green_apple:|
|[PetServiceImpl.java](https://github.com/Gaethering/Gaethering-Server/blob/249c6c96eb36759a8c96bfa6f2ab4d594ffa2dc5/src%2Fmain%2Fjava%2Fcom%2Fgaethering%2Fgaetheringserver%2Fpet%2Fservice%2FPetServiceImpl.java)|78.57%|:x:|
|[PetController.java](https://github.com/Gaethering/Gaethering-Server/blob/249c6c96eb36759a8c96bfa6f2ab4d594ffa2dc5/src%2Fmain%2Fjava%2Fcom%2Fgaethering%2Fgaetheringserver%2Fpet%2Fcontroller%2FPetController.java)|67.5%|:x:|

|Total Project Coverage|48.81%|:x:|
|:-|:-:|:-:|

File: src/main/java/com/gaethering/gaetheringserver/pet/service/PetServiceImpl.java [hunk 1]
```diff
@@ -66,4 +71,40 @@ public boolean setRepresentativePet(String email, Long petId) {
...
+
+    @Override
+    @Transactional
+    public PetRegisterResponse registerPet(String email, MultipartFile multipartFile,
+        PetRegisterRequest petRegisterRequest
+    ) {
+        Member member = memberRepository.findByEmail(email)
+            .orElseThrow(MemberNotFoundException::new);
+
+        if (member.getPets().size() == 3) {
```
Comment (younghoondoodoom) [inline src/main/java/com/gaethering/gaetheringserver/pet/service/PetServiceImpl.java:83]: 저희 2마리까지만 가능한거였나요?? 그리고 3은 상수화 시켰으면 좋겠습니다!

Comment (naekang) [reply src/main/java/com/gaethering/gaetheringserver/pet/service/PetServiceImpl.java:83]: 최대 3마리까지 가능하도록 정했던거 같아요!
상수화 시켜두겠습니다!

**Comment by naekang** [comment]
|File|Coverage [82.01%]|:green_apple:|
|:-|:-:|:-:|
|[PetErrorCode.java](https://github.com/Gaethering/Gaethering-Server/blob/1bfe33183ef4e543dc0f9ecd59f094797b3b9221/src%2Fmain%2Fjava%2Fcom%2Fgaethering%2Fgaetheringserver%2Fpet%2Fexception%2Ferrorcode%2FPetErrorCode.java)|96.3%|:green_apple:|
|[PetServiceImpl.java](https://github.com/Gaethering/Gaethering-Server/blob/1bfe33183ef4e543dc0f9ecd59f094797b3b9221/src%2Fmain%2Fjava%2Fcom%2Fgaethering%2Fgaetheringserver%2Fpet%2Fservice%2FPetServiceImpl.java)|78.57%|:x:|
|[PetController.java](https://github.com/Gaethering/Gaethering-Server/blob/1bfe33183ef4e543dc0f9ecd59f094797b3b9221/src%2Fmain%2Fjava%2Fcom%2Fgaethering%2Fgaetheringserver%2Fpet%2Fcontroller%2FPetController.java)|67.5%|:x:|

|Total Project Coverage|48.81%|:x:|
|:-|:-:|:-:|

**Pull request merged**